data/pages-content/finaid.md

```diff
@@ -40,12 +44,13 @@ to everyone, please consider sponsoring EuroPython by reaching out to
 You can apply for three different types of grants:
 
 
-1. **Free Ticket Voucher Grant:** Get a voucher for a free standard ticket for
+1. **[CLOSED] Free Ticket Voucher Grant:** Get a voucher for a free standard ticket for
    the in-person conference.
-2. **Travel/Accommodation Grant:** We will reimburse travel and/or accommodation
+2. **[CLOSED] Travel/Accommodation Grant:** We will reimburse travel and/or accommodation
    costs up to €400.
-3. **Visa Application Fee Grant:** Get reimbursed for the costs of a short-stay
+3. **[CLOSED] Visa Application Fee Grant:** Get reimbursed for the costs of a short-stay
    Schengen visa to the Czech Republic (up to €80).
+4. **🆕 Free remote ticket:** Get a standard remote ticket join the conference remotely for free.
```

Comment:
Get a standard remote ticket *to* join the conference (...)
or 
Get a standard remote ticket. Join the conference (...)